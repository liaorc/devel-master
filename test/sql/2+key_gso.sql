---#
---#    GpuSort Closed Issue (Remapping of variable) Test Cases
---#    #137, ...
---#

set pg_strom.debug_force_gpusort to on;
set pg_strom.enable_gpusort to on;
set enable_sort to off;
set log_min_messages = debug1;
set client_min_messages to warning;

-- #137
drop table if exists i137;
create table i137(id integer,t1 text collate "C",t2 text collate "C");
insert into i137 select generate_series(1,100) , 'i137','i137';
select * from i137  where id%2=0 order by id,t1,t2;
