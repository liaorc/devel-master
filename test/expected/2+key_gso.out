---#
---#    GpuSort Closed Issue (Remapping of variable) Test Cases
---#    #137, ...
---#
set pg_strom.debug_force_gpusort to on;
set pg_strom.enable_gpusort to on;
set enable_sort to off;
set log_min_messages = debug1;
set client_min_messages to warning;
-- #137
drop table if exists i137;
create table i137(id integer,t1 text collate "C",t2 text collate "C");
insert into i137 select generate_series(1,100) , 'i137','i137';
select * from i137  where id%2=0 order by id,t1,t2;
 id  |  t1  |  t2  
-----+------+------
   2 | i137 | i137
   4 | i137 | i137
   6 | i137 | i137
   8 | i137 | i137
  10 | i137 | i137
  12 | i137 | i137
  14 | i137 | i137
  16 | i137 | i137
  18 | i137 | i137
  20 | i137 | i137
  22 | i137 | i137
  24 | i137 | i137
  26 | i137 | i137
  28 | i137 | i137
  30 | i137 | i137
  32 | i137 | i137
  34 | i137 | i137
  36 | i137 | i137
  38 | i137 | i137
  40 | i137 | i137
  42 | i137 | i137
  44 | i137 | i137
  46 | i137 | i137
  48 | i137 | i137
  50 | i137 | i137
  52 | i137 | i137
  54 | i137 | i137
  56 | i137 | i137
  58 | i137 | i137
  60 | i137 | i137
  62 | i137 | i137
  64 | i137 | i137
  66 | i137 | i137
  68 | i137 | i137
  70 | i137 | i137
  72 | i137 | i137
  74 | i137 | i137
  76 | i137 | i137
  78 | i137 | i137
  80 | i137 | i137
  82 | i137 | i137
  84 | i137 | i137
  86 | i137 | i137
  88 | i137 | i137
  90 | i137 | i137
  92 | i137 | i137
  94 | i137 | i137
  96 | i137 | i137
  98 | i137 | i137
 100 | i137 | i137
(50 rows)

